src/distributions/uniform.rs

```diff
@@ -8,17 +8,17 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! A distribution generating numbers within a given range.
+//! A distribution uniformly generating numbers within a given range.
 
 use Rng;
 use distributions::Distribution;
 use distributions::float::IntoFloat;
 
 /// Sample values uniformly between two bounds.
 ///
-/// `Range::new` and `Range::new_inclusive` will set up a `Range`, which does
+/// `Uniform::new` and `Uniform::new_inclusive` will set up a `Uniform`, which does
```

Comment:
I don't think this sentence read well any more, but have difficulty saying why.

Can you also mention "range" once in this description?

Reply:
You're right. Perhaps something like `U::new` and `U::new_inclusive` construct a `Uniform` distribution sampling from the half-open and closed (inclusive) ranges respectively.

Reply:
I reworded it.